javascript.js

```diff
@@ -243,14 +243,32 @@ seeProject.forEach((button) => button.addEventListener('click', (button) => {
 const contactForm = document.getElementById('contactForm');
 contactForm.addEventListener('submit', (event) => {
   event.preventDefault();
-
   const email = document.getElementById('email').value;
+  const formdata = {
+    email: document.getElementById('email').value,
+    userName: document.querySelector('.user').value,
+    textArea: document.querySelector('.textArea').value,
+  };
   const errorMessage = document.querySelector('.error');
```

Comment:
- [ ] Great job so far 👏🏻 However the input data are preserved in the local storage only when I submit the form. According to the project requirements, When the user changes the content of any input field in the form the data object should be updated and saved to the local storage. even without submitting the form. and make sure
- When the user loads the page, the code must checks if there are any saved data in the local storage, and if so the input fields are pre-filled with the data.